src/main/webapp/scripts/script.js

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// difact_idibuted under the License is difact_idibuted on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//import domtoimage from 'dom-to-image';
+
+
+function login() {
+    fetch('/login_page').then(response => response.json()).then(login => {loginDomManipulation(login.message, login.status);});
```

Comment:
2 block indentation is in the style guide of JavaScript: https://google.github.io/styleguide/jsguide.html#formatting-block-indentation